content/docs/04.5-devx/07-sandbox-clusters.md

```diff
@@ -47,3 +47,57 @@ To create your new sandbox cluster:
    * Review the information and deploy the Palette virtual cluster. The Palette virtual cluster will be provisioned within the next few minutes.
 
 <br />
+
+
+## Palette Virtual Cluster Pause and Resume
+
+The Palette allows the Pause and Resume of Palette Virtual Clusters when not in use. This feature enables the users with optimized resource sharing by pausing the virtual clusters, not in use. This adds significant flexibility in managing operating costs and resource management for the Palette virtual clusters. 
+
+<br />
+
+### Resource Dependencies
+
+* The Quota allocation is independent of a Virtual Cluster's Pause or Resume status.
+
+
+* The CPU and Memory are freed upon pausing a Virtual Cluster.
+
+
+* Resources like Storage and Load Balancers are allocated to the paused Virtual Cluster.
+
+
+* The Apps deployed on a Palette Virtual Cluster goes to a Pause state when the Virtual Cluster is paused.
```

Comment:
```suggestion
* The Apps deployed on a virtual cluster go to a pause state when the cluster is paused.
```
The term virtual cluster has been used previously, so it's fine to stick with it.